ckanext/googleanalytics/reports.py

```diff
@@ -3,28 +3,49 @@
 from datetime import datetime, timedelta
 
 
+def last_week():
+    today = datetime.today()
+    end_date = today - timedelta(days=1)
+    start_date = end_date - timedelta(days=7)
```

Comment:
Should this be days=8, so that the period would be 7 days long instead of 6?

Reply:
And same maybe in month and in year.

Reply:
It counts the timedelta compared to the end_date, which has the one day reduced, so I think it would be 7 days in total?

Reply:
That is true!